src/components/PurchaseOrder/UpdateOrderErrorModal/UpdateOrderErrorModal.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import PropTypes from 'prop-types';
+
+import { Button, Modal } from '@folio/stripes/components';
+
+const UpdateOrderErrorModal = ({ orderNumber, cancel, reason }) => {
+  const footer = (
+    <Button
+      onClick={cancel}
+      data-test-open-order-confirmation-modal-cancel
+      marginBottom0
+    >
+      <FormattedMessage id="ui-orders.openOrderModal.cancel" />
+    </Button>
+  );
+
+  return (
+    <Modal
+      label={<FormattedMessage id="ui-orders.openOrderModal.title" values={{ orderNumber }} />}
+      open
+      data-test-open-order-confirmation-modal
+      footer={footer}
+    >
+      <div data-test-open-order-confirmation-modal-content>
+        <div>
```

Comment:
Do you really need this `<div>`?

Reply:
Good point, thank you. Copy && past it's not a good thing